pipelines/HSC/DRP-RC2.yaml

```diff
@@ -178,19 +175,16 @@ subsets:
       - psfPhotRepStar2
       - psfPhotRepStar3
       - psfPhotRepStar4
-    description: >
-      Set of tasks for calculation of metrics via faro. This includes tasks
-      that operate on matched visits, and thus appear in step3 below.
-  faro_step5:
-    subset:
+      # tract-level on coadd products
       - TE1
       - TE2
       - wPerp
       - skyObjectMean
       - skyObjectStd
     description: >
-      Set of tasks for calculation of metrics via faro. This includes tasks
-      that operate on coadds, and thus appear in step5 below.
+      Set of tasks for calculation of metrics via faro.
```

Comment:
Same comment about adding a sentence saying where in the chain you should (or shouldn't) run it.